src/components/grid-aware/ProgramsBlock/ProgramsBlock.module.css

```diff
@@ -0,0 +1,76 @@
+/* Outter Grid */
+.bleedWrapper {
+  background: var(--color-black);
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
```

Comment:
It looks like in the mobile designs for this component, it's actually a 20px margin on the sides, not 40px.